core/src/main/java/hudson/model/AbstractBuild.java

```diff
@@ -987,7 +987,8 @@ public Calendar due() {
      * {@inheritDoc}
      * The action may have a {@code summary.jelly} view containing a {@code <t:summary>} or other {@code <tr>}.
      */
-    @Override public void addAction(Action a) {
+    @Override
+    public void addAction(Action a) {
```

Comment:
Ah, I forgot about the ancillary formatting changes in the PR.  I will back these out as they are nonessential.